addons/TBMod_arsenal/arsenal/fn_arsenalBW.sqf

```diff
@@ -17,9 +17,9 @@ private _allgemein = [
     // "TFAR_rt1523g_big_rhs",
 
     // Gesichtsbedeckung
-    "BWA3_G_Combat_Clear",
-    "BWA3_G_Combat_Orange",
-    "BWA3_G_Combat_Black",
+    "BWA3_G_Combat_clear",
```

Comment:
Groß und Kleinschreibung macht echt einen Unterschied o.O
Ist das ne Vermutung oder getestet, dann ist das ACE3-Arsenalsystem aber wirklich komisch.

Reply:
Keine Ahnung. Habs über nen Java Programm die Datei getestet. Und da hab ich es so reingepackt wie in der Config